sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/utils/OriginalContent.java

```diff
@@ -116,4 +136,31 @@ private byte[] readHead(Path path) {
             throw LOGGER.logExceptionAsError(new UncheckedIOException(e));
         }
     }
+
+    public Flux<ByteBuffer> convertInputStreamToFluxByteBuffer(InputStream inputStream) {
+        byte[] buffer = new byte[4096];
+        int b;
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        try {
+            while ((b = inputStream.read(buffer)) != -1) {
+                outputStream.write(buffer, 0, b);
+            }
+        } catch (IOException ex) {
+            throw new UncheckedIOException(ex);
+        }
+
+        return Flux.just(ByteBuffer.wrap(outputStream.toByteArray()));
+    }
+
+    public Flux<ByteBuffer> copySeekableByteChannelToFluxByteBuffer(SeekableByteChannel src) throws IOException {
```

Comment:
this would not work for a huge content or would use way too much memory for high-scale testing of a small content.
We should either make it super-optimal or use existing helpers like 
`BinaryData.fromInputStream(Channels.newInputStream(channel))`